### PR TITLE
[feature] aws-single-page-static-site: allow disabling the custom error responses

### PR DESCRIPTION
**Pull request by palasha**
This is useful if we dont want this behavior.

Test Plan: lets see if it works.

### CODE REVIEW

File: aws-single-page-static-site/variables.tf [hunk 1]
```diff
@@ -68,3 +68,10 @@ variable "path_pattern" {
   description = "The pattern (for example, images/*.jpg) that specifies which requests you want this cache behavior to apply to."
   default     = "*"
 }
+
+variable "custom_error_response_codes" {
+  type = list(number)
+  description = "The http response codes for which to return the default index page."
+  default = [404, 403, 502]
```
Comment (alldoami) [inline aws-single-page-static-site/variables.tf:75]:
```suggestion
  default = [404, 403, 503]
```
from what I see I think it's supposed to be `503`?

**Comment by alldoami** [comment]
Also looks like you need to do a `make fmt`

**Comment by palasha** [comment]
- Fixed incorrect response code
- make fmt
- verified this is working as expected on www.comealong.org

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/cztack/compare/40e6d91a05df9206ea596490c3ea341c7ccddbec..7431c50f27c0a6c480965b101a0daa8a5f419e2c).

**Review by ryanking — APPROVED** [review]

**Comment by ryanking** [comment]
@palasha can we merge this?

**Pull request merged**